sn_cli/src/subcommands/files/iterative_uploader.rs

```diff
@@ -104,7 +110,8 @@ fn spawn_progress_handler(
 ) -> JoinHandle<Result<(), Error>> {
     tokio::spawn(async move {
         let mut upload_terminated_with_error = false;
-        // The loop is guaranteed to end, as the channel will be closed when the upload completes or errors out.
+        // The loop is guaranteed to end, as the channel will be
```

Comment:
Really minor issue, but your line length seems a bit small for these comments. I think `rustfmt` uses 100 as a line length, which is a bit more modern than typical values like 72 or 80. I think that's also a decent line length for comments and doc comments.

Reply:
resolved